[0m16:02:32.665140 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025CAAD15A90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025CA8EE9310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025CABE47D90>]}


============================== 16:02:32.675204 | dae8210c-1414-4184-84a7-22fa9c3789d9 ==============================
[0m16:02:32.675204 [info ] [MainThread]: Running with dbt=1.9.2
[0m16:02:32.677068 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\kship\\.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt init', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:02:42.662101 [debug] [MainThread]: Starter project path: C:\python3-13\Lib\site-packages\dbt\include\starter_project
[0m16:02:42.719374 [info ] [MainThread]: 
Your new dbt project "cpde" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:02:42.722290 [info ] [MainThread]: Setting up your profile.
[0m16:04:30.355685 [info ] [MainThread]: Profile cpde written to C:\Users\kship\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:04:30.358685 [debug] [MainThread]: Command `dbt init` succeeded at 16:04:30.358357 after 117.95 seconds
[0m16:04:30.361314 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025CCBCB2780>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025CABED7AD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025CCBC86470>]}
[0m16:04:30.362351 [debug] [MainThread]: Flushing usage events
[0m16:04:31.640268 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m16:09:42.526446 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255CEF55A90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255CD129310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255D0077D90>]}


============================== 16:09:42.535795 | 47802600-dd93-4ce3-ba15-90b774203009 ==============================
[0m16:09:42.535795 [info ] [MainThread]: Running with dbt=1.9.2
[0m16:09:42.537320 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\kship\\.dbt', 'version_check': 'True', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:09:42.577945 [info ] [MainThread]: dbt version: 1.9.2
[0m16:09:42.580002 [info ] [MainThread]: python version: 3.13.2
[0m16:09:42.581249 [info ] [MainThread]: python path: C:\python3-13\python.exe
[0m16:09:42.582158 [info ] [MainThread]: os info: Windows-11-10.0.22631-SP0
[0m16:09:43.107544 [info ] [MainThread]: Error importing adapter: No module named 'dbt.adapters.mysql'
[0m16:09:43.109830 [info ] [MainThread]: Error importing adapter: No module named 'dbt.adapters.mysql'
[0m16:09:43.113174 [info ] [MainThread]: Using profiles dir at C:\Users\kship\.dbt
[0m16:09:43.115011 [info ] [MainThread]: Using profiles.yml file at C:\Users\kship\.dbt\profiles.yml
[0m16:09:43.115785 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\kship\Desktop\Capstone Project - Data Engineering\dbt_project.yml
[0m16:09:43.116786 [info ] [MainThread]: Configuration:
[0m16:09:43.117553 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m16:09:43.118240 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m16:09:43.118974 [info ] [MainThread]: Required dependencies:
[0m16:09:43.119827 [debug] [MainThread]: Executing "git --help"
[0m16:09:43.164783 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m16:09:43.165450 [debug] [MainThread]: STDERR: "b''"
[0m16:09:43.165913 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m16:09:43.166924 [info ] [MainThread]: Connection test skipped since no profile was found
[0m16:09:43.168013 [info ] [MainThread]: [31m2 checks failed:[0m
[0m16:09:43.168958 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  Credentials in profile "dbt_mysql_new", target "dev" invalid: Runtime Error
    Could not find adapter type mysql!

Runtime Error
  Credentials in profile "dbt_nq", target "dev" invalid: Runtime Error
    Could not find adapter type mysql!


[0m16:09:43.171650 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\kship\Desktop\Capstone Project - Data Engineering\dbt_project.yml> not found

[0m16:09:43.173410 [debug] [MainThread]: Command `dbt debug` failed at 16:09:43.173229 after 0.89 seconds
[0m16:09:43.173914 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255EFF78510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255EFF8C710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000255EFE6F130>]}
[0m16:09:43.174448 [debug] [MainThread]: Flushing usage events
[0m16:09:44.193237 [debug] [MainThread]: An error was encountered while trying to flush usage events
